cmake/AndroidManifest.xml.in

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0"?>
-<manifest package="ch.opengis.qfield" xmlns:android="http://schemas.android.com/apk/res/android" android:versionName="%%VERSION_NAME%%" android:versionCode="%%VERSION_CODE%%" android:installLocation="auto">
```

Comment:
I don't have an idea, but just noticed. Why we used here always "qfield" and with this change we use "qfield" or "qfield_dev" or "qfield_beta"? Why this has to be changed? 
@m-kuhn